cmd/taskhandler/main.go

```diff
@@ -52,9 +51,12 @@ func serveCache() func() error {
 func serveProxy() {
 
 	var (
-		metricsPath = viper.GetString("metrics.metricsPath")
-		restPort    = viper.GetInt("proxyRestPort")
-		grpcPort    = viper.GetInt("proxyGrpcPort")
+		restPort = viper.GetInt("proxyRestPort")
+		restHost = viper.GetString("serving.restHost")
```

Comment:
Can we name this variable `servingRestHost` to distinguish it from the proxy port

Reply:
done